extensions/amp-consent/0.1/consent-config.js

```diff
@@ -151,6 +151,18 @@ export class ConsentConfig {
         }
       }
     }
+
+    // Migrate to geoOverride
+    const group = config['promptIfUnknownForGeoGroup'];
+    if (group) {
+      config['consentRequired'] = false;
+      config['geoOverride'] = {
+        [group]: {
+          'consentRequired': true,
+        },
+      };
+    }
+
```

Comment:
then you can assert `config['consentRequired']` should not be `undefined` in `validateMergedGeoOverride_`